spec/helpers/nested_attributes_for_strategy.rb

```diff
@@ -0,0 +1,42 @@
+class NestedAttributesForStrategy
```

Comment:
Delete this file, it was a good run. Should use the regular old `attributes_for` strategy and manually add the nested attributes where necessary.